bigtable/cmd/cbt/cbt.go

```diff
@@ -420,7 +420,7 @@ var commands = []struct {
 			"  batch-size=<500>                      The max number of rows per batch write request\n" +
 			"  workers=<1>                           The number of worker threads\n\n" +
 			"  Import data from a csv file into an existing cbt table that has the required column families.\n" +
-			"  See <example.csv.github.com/cbt-import-sample.csv> for a sample .csv file and formatting.\n" +
+			"  See <https://github.com/googleapis/google-cloud-go/blob/main/bigtable/exampledata/cbt-import-data.csv> for a sample .csv file and formatting.\n" +
```

Comment:
Before the "see file" sentence can we just describe the format a little? This is what I'm putting in the blog post to describe it `The CSV file uses one row of headers specifying the column qualifiers and a blank for the rowkey. You can add an additional row of headers for the column families and then remove the column-family argument from the import command.`